shared/idmap/idmapset_linux.go

```diff
@@ -641,22 +641,27 @@ func (set *IdmapSet) doUidshiftIntoContainer(dir string, testmode bool, how stri
 			// Dump capabilities
 			if fi.Mode()&os.ModeSymlink == 0 {
 				caps, err = GetCaps(path)
-				if err != nil {
-					return err
+				switch err {
+				case syscall.EOPNOTSUPP:
+					logger.Warnf("llistxattr is not supported on %s", path)
+				case nil:
+					// noop
+				default:
+					return errors.Wrapf(err, "failed getting caps on %s: %v", path, err)
 				}
 			}
 
 			// Shift owner
 			err = ShiftOwner(dir, path, int(newuid), int(newgid))
 			if err != nil {
-				return err
+				return errors.Wrapf(err, "failed shifting %s %s %d %d: %v", dir, path, int(newuid), int(newgid), err)
```

Comment:
Please can you change this to "Failed..." to be consistent with other LXD error messages. Thanks

Reply:
I updated the error messages, but I'm actually not sure what style you want.  I made them more succinct, not bothering to say 'failed'.  If you're after something different, please let me know.

Reply:
We generally do "Failed ..." (upper case F). Thanks